spec/views/bulk_actions/new.html.erb_spec.rb

```diff
@@ -5,11 +5,20 @@
 RSpec.describe 'bulk_actions/new.html.erb' do
   let(:user_groups) { %w[dlss-developers top-secret-clearance] }
   let(:apo_list) { [['APO 1', 'druid:123'], ['APO 2', 'druid:234']] }
+  let(:current_user) { double(sunetid: 'esnowden', groups: user_groups) }
+  let(:collections) do
+    [
+      ['None', ''],
+      ['Dummy Collection 1', 'druid:123'],
+      ['Dummy Collection 2', 'druid:456']
+    ]
+  end
 
   before do
     @form = BulkActionForm.new(create(:bulk_action), groups: user_groups)
-    allow(view).to receive(:current_user).and_return(double(sunetid: 'esnowden', groups: user_groups))
+    allow(view).to receive(:current_user).and_return(current_user)
     allow(view).to receive(:apo_list).with(user_groups).and_return(apo_list)
+    allow(current_user).to receive(:permitted_collections).and_return(collections)
```

Comment:
If you add this to the double on line 8, then you don't need this line.

Reply:
Done